### PR TITLE
Only forward metric log to logstash

### PR DESCRIPTION
**Pull request by coderla**
Current metrics log make console of registry very hard to read another log, so only forward metric log to logstash, not log metrics to console. This make easier when need to check registry error.

- Please make sure the below checklist is followed for Pull Requests.

- [x] [Travis tests](https://travis-ci.org/jhipster/jhipster-registry/pull_requests) are green
- [x] Tests are added where necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-registry/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by PierreBesson** [comment]
@coderia There is code to do something like you do but better on the main generator (by marking metrics logs with a `metric` marker. Your PR just made me realize that the registry is not up to date with the generator. It's even using udp instead of TCP to report log.
Would you mind amending this PR to make the LoggingConfiguration identical to the one on normal JHipster app so the problem will be correctly fixed.

**Comment by coderla** [comment]
@PierreBesson Yes, I will try to get the latest code of LoggingConfiguration in main generator and make a another pull. So the LoggingConfiguration will identical with the latest one.

File: src/main/java/io/github/jhipster/registry/config/LoggingConfiguration.java [hunk 1]
```diff
@@ -25,63 +40,91 @@
 
     private final String serverPort;
 
-    private final String instanceId;
```
Comment (PierreBesson) [inline src/main/java/io/github/jhipster/registry/config/LoggingConfiguration.java]: This is strange, why remove instanceId ? This is needed for microservices.

Comment (coderla) [reply src/main/java/io/github/jhipster/registry/config/LoggingConfiguration.java]: This is the code that generated from main generation. Just make sure it's identical with another microservices.

Comment (PierreBesson) [reply src/main/java/io/github/jhipster/registry/config/LoggingConfiguration.java]: You are right ! It seems it has been removed at some point 😢 
It used to be here https://github.com/jhipster/generator-jhipster/commit/7993651ec2e80c97c7d8cf3b9f636e0b0522eb1a
I need to put this back as is is essential for the console.

Comment (coderla) [reply src/main/java/io/github/jhipster/registry/config/LoggingConfiguration.java]: So if that, you will need to update the generator to bring the instanceId back.

Comment (coderla) [reply src/main/java/io/github/jhipster/registry/config/LoggingConfiguration.java]: Should we merge this request or waiting to the generator update?

Comment (PierreBesson) [reply src/main/java/io/github/jhipster/registry/config/LoggingConfiguration.java]: Agreed. I'm merging this. Thanks a lot for your time and congrats on your first contribution.
I changed your PR to point to the develop branch.

Comment (coderla) [reply src/main/java/io/github/jhipster/registry/config/LoggingConfiguration.java]: Thank @PierreBesson, will do that in the next time.

**Pull request merged**